src/Components/Modal/Modal.css

```diff
@@ -40,4 +40,11 @@
 .close-Btn:hover {
     cursor: pointer;
     color: red;
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 600px) {
+    .modal-main {
+        width: 90%;
+    }
+}
```

Comment:
I think we'll want to add another breakpoint around 300px.  It looks great on everything but the Fold view and then it gets a little weird.